src/steps/check-lead-activity.ts

```diff
@@ -156,6 +182,17 @@ export class CheckLeadActivityStep extends BaseStep implements StepInterface {
 
     return result;
   }
+
+  createRecords(activities) {
+    const records = [];
+    activities.forEach((activity) => {
+      activity.attributes.forEach(attr => activity[attr.name] = attr.value);
+      records.push(activity);
+    });
+    const headers = { id: 'ID', leadId: 'Lead ID', activityDate: 'Activity Date', activityTypeId: 'Activity Type ID' };
+    activities[0].attributes.forEach(attr => headers[attr.name] = attr.name);
```

Comment:
- [ ] Perhaps the table header value could be title-cased?  It looks odd next to the title-case of the default/existing headers.  ...And it looks like it's inconsisntent based on the type of activity too.

![Screen Shot 2020-02-13 at 7 18 35 PM](https://user-images.githubusercontent.com/3496491/74498733-73052780-4e96-11ea-8dcf-8d0abcd14b6a.png)

https://www.npmjs.com/package/title-case

```javascript
headers[attr.name] = titleCase(attr.name);
```

Reply:
Oh wow, I created my own `titleCase` function after finding your comment below

Reply:
before**

Reply:
@amcpanaligan I'm still seeing lower-case headers when I run the `crank run test/scenarios/Lead\ Activity.crank.yml --debug` locally with the latest code.

Reply:
Yeah. Ended up using the `title-case` package. *pushing..

Reply:
Pushed